README.md
# scenario
 This is a booking system for events and shows


CHANGE_LOG.txt
------------------------------------------------------------
// Abril 15, 2015

- Primer Dia del proyecto.
- Organizacion de Datos.
- Organizacion de Directorios
- Investigacion.
- Maquetacion.
- Creacion de Login y Logout.
------------------------------------------------------------
------------------------------------------------------------
// Abril 16, 2015

- Creacion de Base de Datos: Scenario
- Creacion de Tabla: global_users
- Registro de Usuario: signup.php
- Creado registro y sesión de usuarios.

------------------------------------------------------------
------------------------------------------------------------
// Abril 17, 2015

- Investigacion y Planificacion
- Pruebas
------------------------------------------------------------
------------------------------------------------------------
// Abril 18, 2015

- Creacion de Interfaz de añadir escenario
- Pruebas.
- Diseño de Matriz de Escenario.
------------------------------------------------------------
------------------------------------------------------------
// Abril 19, 2015
- Diseño de Matriz de Escenario (Continuación).
- Diseño de Asientos.
- Diseño de Mesas.
------------------------------------------------------------
------------------------------------------------------------
// Abril 20, 2015
- Trabajos en Base De Datos.
- Creacion Tabla "global_scenarios"
------------------------------------------------------------
------------------------------------------------------------
// Abril 21, 2015
- Trabajos en Base De Datos.
- Programacion y Modificacion de de tabla "global_scenarios"
- Disponiblilidad para guardar informacion Principal de Escenarios
- Modificacion a la seccion "Escenarios".
- Creacion de la Barra Superior de Navegacion "toolbar_nav.php"
- Estilos para la Barra Superior de Navegacion
- Creado events.php
- Creado settings.php
- Creado tickets.php
------------------------------------------------------------
------------------------------------------------------------
// Abril 22, 2015
- Modificaciones de Diseño Grafico.
- Construccion del Motor Light Enginer 0.1v (Fase Gamma)
- Modificacion al Diseño del Login principal.
- Modificacion a la hoja de Estilos.
- Creada tabla "global_areas"
- Modificada tabla "global_structure"
- Creando la modalidad de guardar la estructura del escenario.
- Se pueden guardar el numero de filas y sillas.
- Programando Seleccion de Areas.
------------------------------------------------------------
------------------------------------------------------------
// Abril 23, 2015
- Revision de Sistema.
- Creado directorio "/admin"
- Creado directorio "/media/images"
------------------------------------------------------------
------------------------------------------------------------
// Abril 24, 2015
- Modificacion de Estilos CSS
- Login Centrado.
- Insertando Funciones JavaScript para mejorar formularios.
- Creando funcion para implementar Areas.
------------------------------------------------------------
------------------------------------------------------------
// Abril 25, 2015
- Creada Copia de Seguridad en la nube.
- Revision de Sistema y Correcion de Detalles.
- Pruebas y Ajustes.
------------------------------------------------------------
------------------------------------------------------------
// Abril 26, 2015
- Revision de Sistema y Correcion de Detalles.
- Correccion en Sistema de Areas, Error "Undefined Offset" Eliminado
- Agregado nuevo Sistema de Ayuda.
- Agregado sistema de funciones "light-common"
- Creada tabla "global_chairs"
- Modificada tablar "global_areas"
- Añadidos nuevos parametros para guardar estructura de escenario.
------------------------------------------------------------
------------------------------------------------------------
// Abril 27, 2015
- Revision de Sistema y Correcion de Detalles.
- Modificaciones Menores
------------------------------------------------------------
------------------------------------------------------------
// Abril 28, 2015
- Creando modelo esqueleto de Areas de Escenario
- Revision de Sistema y Correcion de Detalles.
------------------------------------------------------------
------------------------------------------------------------
// Abril 29, 2015
- Revision de Sistema y Correcion de Detalles.
- Creada tabla "global_events"
------------------------------------------------------------
------------------------------------------------------------
// Abril 30, 2015
- Revision de Sistema y Correcion de Detalles.
- Revision y Correccion a la Base de Datos.
- Añadiendo Informacion y campos a la Base de Datos.
- Modificada tabla "global_users"
- Creada de nuevo "global_areas"
- Creada de nuevo "global_chairs"
- Creada de nuevo "global_events"
- Modificada tabla "global_structure"
- Creada tabla "global_configuration"
- Creada tabla "global_tickets"
- Cambiado a Motor InnoDB
------------------------------------------------------------
// Mayo 01, 2015
- Creada Copia de Seguridad en la nube.
- Revision de Sistema y Correcion de Detalles.
------------------------------------------------------------
// Mayo 02, 2015
- Revision de Sistema y Correcion de Detalles.
- Creada tabla "global_admin"
- Modificada "global_configuration"
- Creado "load_config.php"
- Construyendo framework del motor Light Enginer
- Creado manejador de contenido "content.php"
------------------------------------------------------------
// Mayo 03, 2015
- Revision de Sistema y Correcion de Detalles.
- Construcción del Motor Light Enginer.
- Creacion de Defines.
- Creacion de Framework
- Creado "light-html"
- Creada estructura PHP para HTML.
- Creado diseño HTML5.
- Configurado diseño adaptivo HTML5.
- Modificaciones a la Base de Datos.
- Modiciaciones a los estilos principales.
- Conformando programacion abstracta.
- Conformando bibliotecas y archivos del Motor Light Enginer

enginer/config.php
<?php /*----------------------------------------------------------------------------------
    (C) 2015 Something Special | DoItSpecial.com | soporte@doitspecial.com
    config.php | v 1.0 | date: 02/07/2015
------------------------------------------------------------------------------------------
    Configuration File System
----------------------------------------------------------------------------------------*/
    if (stristr(htmlentities($_SERVER['PHP_SELF']),'config.php')) { header('Location: index.php'); die(); }
//----------------------------------------------------------------------------------------
    // SQL Configuration
	$dbhost = 'localhost';                  // SQL Host: localhost
	$dbname = 'scenario';                   // SQL Database Name: test
	$dbpass = '';                           // SQL Password: 12345
	$dbuser = 'root';                       // SQL User: root
   	$dbtype = 'mysql';                      // SQL Database TypeTipo: mysql, mysqli
    $prefix = 'global';                     // SQL Database Prefix: global
//----------------------------------------------------------------------------------------
    // SQL Conecction
    $sql_on = mysql_connect($dbhost, $dbuser, $dbpass);
    if(!$sql_on){ include(PATH_HTML.DS.'global-offline_content'.PHP); }

    $sql_link = mysql_select_db($dbname, $sql_on);
    if (!$sql_link) { die ('MySQL Error: ' . mysql_error()); }

    $mysqli = new mysqli($dbhost, $dbuser, $dbpass, $dbname);
    if ($mysqli->connect_errno) { include(PATH_HTML.DS.'global-offline_content'.PHP); }
//----------------------------------------------------------------------------------------
    // Web Configuration
    $web_config = $mysqli->query("SELECT * FROM ".$prefix."_configuration WHERE id = '1'");
    $web = $web_config->fetch_assoc();

    $dirw     = 'http://localhost/scenario/'; // Web URL: localhost
    $content  =  $web['content_manager'];       // Content Manager Used
    $content2 =  $web['content_admin'];         // Content Manager to Administration Used
    $get_id   =  $web['php_get'];               // PHP Get Used
    $lang     =  $web['languague'];             // Display Language
    $theme    =  $web['theme'];                 // Display Theme
    $setup    =  $web['setup_complete'];        // Setup Status
//----------------------------------------------------------------------------------------
//  PHP Check Version

    $GLOBALS['required_php_version'] = '5.0.4';
    $phpversion = phpversion();

	if($phpversion < $GLOBALS['required_php_version']){
	echo "La versión de PHP en tu servidor es inferior a 5.0.4, <br>la web no funciona en versiones anteriores a esta";
	exit;}
//----------------------------------------------------------------------------------------
//  Configuration

    class Config {

//  Light WebSite Config

    var $offline = 0;
    var $offline_message = 'Disculpa la web esta OFFLINE. Volveremos dentro de pocos instantes...';
    var $editor = 'tinymce';
    var $sitename = 'MasterTicket';
    var $list_limit = '20';
    var $legacy = '0';
    var $MetaDesc = 'Cuando piensas en Tickets, piensa en MasterTicket. Somos tu solución idea para entradas y ventas OnLine.';
	var $MetaKeys = 'MasterTicket, Master+Ticket, master, entradas, tickets, eventos, espectaculos, teatro, musica';
    var $MetaTitle = '1';
    var $MetaAuthor = '1';
    var $short_sitename = 'MasterTicket';

//  Light Debug Config

    var $debug = '1';
    var $debug_lang = '1';

//  Light SQL Layer Config

    var $dbtype = 'mysql';
    var $host = 'localhost';
    var $user = 'root';
    var $password = '';
    var $db = 'scenario';
    var $dbprefix = 'global';

//  Light Server Config
    var $live_site = '';
    var $secret = 'umfdADec1fvTXhUn';
    var $gzip = 1;
    var $error_reporting = 0; // None: 0 / Default: -1 / Simple: 7 / Maxim: 2047 /
    var $helpurl = 'light/';
    var $xmlrpc_server = 1;
    var $ftp_host = '';
    var $ftp_port = '21';
    var $ftp_user = '';
	var $ftp_pass = '';
	var $ftp_root = 'www';
    var $ftp_enable = '1';
    var $force_ssl = '1';
    var $offset = '0';
    var $offset_user = '0';

//  Light Email Config
    var $mailer = 'mail';
    var $mailfrom = 'contacto@masterticketca.com';
    var $fromname = 'MasterTicker';
    var $sendmail = '/usr/sbin/sendmail';
    var $smtpauth = '0';
    var $smtpsecure = '1';
	var $smtpport = '465';
	var $smtpuser = '';
    var $smtppass = '';
    var $smtphost = 'host.caracasemail.com';

//  Light Cache Config
    var $caching = 1;
    var $cachetime = '15';
    var $cache_handler = 'file';
    var $memcache_settings = array();

        //  SEO //
    var $sef = '1';
    var $sef_rewrite = '1';
    var $sef_suffix = '1';

        // Configuraciones de Feed //
    var $feed_limit = 10;
    var $feed_email = 'author';
    var $log_path =  'enginer/logs';
    var $tmp_path = 'enginer/tmp';

        // Configuraciones de Sessión //
	var $lifetime = '15';
	var $session_handler = 'database';

//  Light Enginer Config

    var $app_version = '1.0';
    var $php_web_version = '5.3.1';
    var $forum_version = '4.0';
    var $theme_enginer = 'light';
    var $mode_enginer = 'light-speed-(beta-mode)';
    var $act_percent = '100.00';
    var $enginer_lang = 'es';
    var $author = 'Something Special';
    var $generator = 'Light';
    var $distribution = 'global';
    var $rating = 'general';
    var $country = 'Venezuela';   }

class eFrameworkConfig
{
	var $dbtype 	= 'mysql';
	var $host 		= 'localhost';
	var $user 		= 'doitspec_admin';
	var $password 	= '23418878legl';
	var $db 		= 'doitspec_global';
	var $dbprefix 	= 'global_';
	var $ftp_host 	= 'host.caracasemail.com';
	var $ftp_port 	= '21';
	var $ftp_user 	= '';
	var $ftp_pass 	= '';
	var $ftp_root 	= '';
	var $ftp_enable = 0;
	var $tmp_path	= '/tmp';
	var $log_path	= '/logs';
	var $mailer 	= 'mail';
	var $mailfrom 	= 'contacto@doitspecial.com';
	var $fromname 	= '';
	var $sendmail 	= '/usr/sbin/sendmail';
	var $smtpauth 	= '0';
	var $smtpsecure = 'none';
	var $smtpport	= '465';
	var $smtpuser 	= '';
	var $smtppass 	= '';
	var $smtphost 	= 'host.caracasemail.com';
	var $debug 		= 0;
	var $caching 	= '0';
	var $cachetime	= '900';
	var $language  	= 'es-ve';
	var $secret		= null;
	var $editor		= 'none';
	var $offset		= 0;
	var $lifetime	= 15;
} ?>